requirements-devel.txt

```diff
@@ -1,22 +1,24 @@
 astroid==2.13.2
 attrs==22.2.0
 black==22.12.0
+cachetools==5.2.0
 catkin-pkg==0.5.2
-certifi==2022.12.7
+certifi==2022.9.24
```

Comment:
Are these "downgrades" are just from the need to rebase the PR?

Reply:
Other than `snap-helpers` and `craft-providers` I think they're all related to specifications from tox - certainly `certifi` is. Once we get renovate everywhere this should be more reliable.